<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QSlider" name="przesunX_Slider">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="przesunY_Slider">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="przesunZ_Slider">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="obrotZ_Slider">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>100</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="obrotY_Slider">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>70</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="obrotX_Slider">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Przesunięcia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Obrót</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Skalowanie</string>
    </property>
   </widget>
   <widget class="QSlider" name="skal_Slider">
    <property name="geometry">
     <rect>
      <x>589</x>
      <y>170</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>250</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pochylenia</string>
    </property>
   </widget>
   <widget class="QSlider" name="pochylX_Slider">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>280</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="pochylY_Slider">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="pochylZ_Slider">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>340</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_czas">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>380</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>460</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>240</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>teksturowanie</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_X">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>430</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>oś X</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Y">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>430</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>oś Y</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>260</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kontury</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
